src/java/com/puppetlabs/jruby_utils/pool/LockablePool.java

```diff
@@ -122,7 +122,12 @@ E borrowItemWithTimeout(long timeout, TimeUnit unit)
      */
     int currentSize();
 
-   /**
+    /**
+     * Returns the number of jruby instances in the pool.
```

Comment:
"...number of unused jruby instances..."

Reply:
whoops, will fix